{
  "test_type": "unit",
  "timestamp": "2025-07-27T19:33:17.291938",
  "bug_id": "mrr_api_misuse_0364",
  "results": {
    "total": 63,
    "passed": 49,
    "failed": 4,
    "skipped": 4
  },
  "failures": [
    {
      "test": "test_api_misuse_0",
      "error": "429 errors",
      "file": "src/core/manager_60.py",
      "line": 187
    },
    {
      "test": "test_api_misuse_1",
      "error": "Requests blocked",
      "file": "src/core/manager_60.py",
      "line": 83
    }
  ]
}